executor/builder.go

```diff
@@ -260,51 +278,79 @@ func (b *executorBuilder) buildIndexScan(v *plan.IndexScan) Executor {
 		} else {
 			ex = b.buildUnionScanExec(e)
 		}
+		ex = b.buildFilter(ex, v.AccessConditions)
```

Comment:
yes.
it's a mistake that I confused "use prefix index or not" with "truncate data or not".